src/summon/browzine-summon-adapter.js

```diff
@@ -185,8 +185,8 @@ browzine.summon = (function() {
 
   function directToPDFTemplate(directToPDFUrl) {
     var pdfIcon = "https://assets.thirdiron.com/images/integrations/browzine-pdf-download-icon.svg";
-    var articlePDFDownloadWording = browzine.articlePDFDownloadWording || "Article PDF";
-    var articlePDFDownloadLinkText = browzine.articlePDFDownloadLinkText || "Download Now";
+    var articlePDFDownloadWording = browzine.articlePDFDownloadWording || browzine.summonArticlePDFDownloadWording || "Article PDF";
+    var articlePDFDownloadLinkText = browzine.articlePDFDownloadLinkText || browzine.summonArticlePDFDownloadLinkText || "Download Now";
```

Comment:
Is there a test for the `summonArticlePDFDownloadWording` ?

Reply:
Nope. Will add.